index.bs

```diff
@@ -263,6 +263,30 @@ so that it works across screen sizes,
 and allows users to choose their own font and font size without causing text to overflow.
 </div>
 
+<h3 id="backwards-compat">Support Existing Content</h3>
+
+Existing content often relies upon expected user agent processing and behavior to function as intended.
+Processing requirements should be specified to ensure that user agents will be able to handle most existing content.
+In particular, it should be possible to process existing web content in current user agents and get results that are compatible with the existing expectations of users and authors,
+based on the behavior of existing browsers.
+
+Content relying on existing browser behavior can take many forms.
+It may rely on elements, attributes or APIs that are part of earlier specifications, or on features that are entirely proprietary.
+It may depend on specific error handling rules. It may depend on certain method names not being defined.
+In rare cases, it may even depend on a feature from earlier specifications not being implemented as specified.
+
+When considering changes to legacy features or behavior, relative to current implementations and author expectations, the following questions should be considered:
+
+* Does a significant quantity of existing content depend on the feature or behavior?
+* Does any of the dependent content occur on particularly popular websites?
+* Is the dependent content genuinely intended for consumption, rather than occurring solely in test cases or examples?
+* Is the dependent content on the public web, rather than found solely on internal sites with a controlled user environment?
+* Does the dependent content currently work as intended in multiple popular user agents, rather than explicitly targeting only one particular user agent?
+
+The benefit of the proposed change should be weighed against the likely cost of breaking content, as measured by these criteria.
+In some cases, it may be desirable to make a nonstandard feature or behavior part of the web platform, if it satisfies a valid use case.
+However, the fact that something is part of the web platform does not by itself mean that relying on it is condoned or encouraged.
```

Comment:
Yes sometimes it doesn't make sense to make that change, but other times it does, what matters is making a public plan for the change with a roadmap and time line so that people and companies can prepare and adjust